windows-lb/compute.tf

```diff
@@ -15,19 +25,22 @@ resource "oci_core_instance" "instance1" {
 
   create_vnic_details {
     subnet_id      = oci_core_subnet.subnet1.id
-    hostname_label = "be-instance1"
+    hostname_label = "ASPNetVM1"
   }
 
+  fault_domain = "FAULT-DOMAIN-1"
+
   source_details {
     source_type = "image"
     source_id   = data.oci_core_images.this.images[0].id # var.instance_image_ocid[var.region]
   }
+  defined_tags = { "${oci_identity_tag_namespace.ArchitectureCenterTagNamespace.name}.${oci_identity_tag.ArchitectureCenterTag.name}" = var.release }
 }
 
 resource "oci_core_instance" "instance2" {
-  availability_domain = random_shuffle.compute_ad.result[2 % length(random_shuffle.compute_ad.result)]
+  availability_domain = var.availablity_domain_name == "" ? data.oci_identity_availability_domains.ADs.availability_domains[0]["name"] : var.availablity_domain_name
```

Comment:
Same as the comment for the other instance.

Reply:
@jeevanjoseph the answer as above.

Reply:
@jeevanjoseph after a call I will revert it back to random_shuffle.